src/User.php

```diff
@@ -642,7 +642,7 @@ public function getFromDBbyEmail($email, $condition = [])
     public function getDefaultEmail()
     {
 
-        if (!isset($this->fields['id'])) {
```

Comment:
This change has no impact on your test, so it may not be necessary.

Reply:
On some versions of PHP and mariaDB, this code would generate an error on users where the id was equal to an empty string. Using the `empty()` function avoids this error.